packages/webapp/pages/helloworld/index.tsx

```diff
@@ -99,6 +101,14 @@ export default function HelloWorldPage({
   const { funnel, session } = funnelBoot?.funnelState ?? {};
   const { isAuthReady, isValidRegion, user } = useAuthContext();
   const router = useRouter();
+  const { setTheme, themeMode } = useSettingsContext();
+
+  useEffect(() => {
+    const theme = funnel?.parameters?.theme?.mode;
+    if (!isServer && !!theme && theme !== themeMode) {
```

Comment:
Just to make sure we are checking at the right time.
```suggestion
    if (!isServer && isAuthReady && !!theme && theme !== themeMode) {
```